app/uk/gov/hmrc/vatapi/resources/wrappers/FinancialDataResponse.scala

```diff
@@ -63,7 +63,11 @@ case class FinancialDataResponse(underlying: HttpResponse) extends Response {
     case 400 if errorCodeIsOneOf(INVALID_VRN) => BadRequest(toJson(Errors.VrnInvalid))
     case 400 if errorCodeIsOneOf(INVALID_DATEFROM) => BadRequest(toJson(Errors.InvalidDateFrom))
     case 400 if errorCodeIsOneOf(INVALID_DATETO) => BadRequest(toJson(Errors.InvalidDateTo))
-    case 400 if errorCodeIsOneOf(NOT_FOUND) => NotFound(toJson(Errors.NotFound))
+    case 400 if errorCodeIsOneOf(INVALID_IDTYPE, INVALID_IDNUMBER, INVALID_ONLYOPENITEMS, INVALID_REGIMETYPE, INVALID_INCLUDELOCKS,
```

Comment:
Once again INVALID_IDNUMBER would map onto a 400 we would return back to the vendor, RAML snippet below

Invalid VRN | 400 (Bad Request) | VRN_INVALID